custom_components/tuya_local/devices/zym201_presence_sensor.yaml

```diff
@@ -4,7 +4,7 @@ products:
     name: ZY-M201-WIFI24G
 primary_entity:
   entity: binary_sensor
-  class: occupancy
+  class: presence

```

Comment:
The way I understand the difference between these two classes is that occupancy is tied to rooms, and presence is tied to people.  I think occupancy is correct in that case, as we cannot identify an individual who is present.